src/common/users.ts

```diff
@@ -546,8 +546,9 @@ export const shouldAllowRestore = async (
 ) => {
   const { userId, lastViewAt: lastViewAtDb } = streak;
   const user = await con.getRepository(DbUser).findOneBy({ id: userId });
-  const today = new Date();
-  const lastView = new Date(lastViewAtDb);
+  const timezone = user.timezone || 'utc';
```

Comment:
we have `DEFAULT_TIMEZONE` const probably better to use it

Reply:
Oh, nice, yeah. Let's use that.